src/main/java/Duke.java

```diff
@@ -1,10 +1,185 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    static ArrayList<Task> taskList = new ArrayList<>();
+
+    private static Task parseTask(String[] args) {
+        String description;
+        Task newTask = new Task();
+        switch (args[0]) {
+            case "deadline":
+                int byPosition = Arrays.asList(args).indexOf("/by");
+                // TODO if not found
+                description = String.join(" ", Arrays.copyOfRange(args, 1, byPosition));
+                String by = String.join(" ", Arrays.copyOfRange(args, byPosition + 1, args.length));
+                newTask = new Deadline(description, by);
+                break;
+            case "event":
+                int atPosition = Arrays.asList(args).indexOf("/at");
+                description = String.join(" ", Arrays.copyOfRange(args, 1, atPosition));
+                String at = String.join(" ", Arrays.copyOfRange(args, atPosition + 1, args.length));
+                newTask = new Event(description, at);
+                break;
+            case "todo":
+                description = String.join(" ", Arrays.copyOfRange(args, 1, args.length));
+                newTask = new Todo(description);
+                break;
+        }
+        return newTask;
+    }
+
+    private static InputHandleStatus handleOneInputLine(String line) {
+        String[] splitted = line.split("\\s+");
```

Comment:
Should this variable be plural, since its an array?

Reply:
`splitteds` sounds weird :thinking: Any suggestions?

Reply:
maybe something like commandParams?